public/data/ahnafahamed.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Ahnaf Ahamed",
+  "bio": "Front-end developer",
+  "avatar": "https://avatars.githubusercontent.com/u/42427928?v=4",
```

Comment:
Change Avatar to `https://github.com/AhnafAhamed.png` for auto updating.

Reply:
I'll do that, but when I do that I can't see my profile picture. Please enlighten me. Thanks